images/sdn/Dockerfile.rhel

```diff
@@ -1,16 +1,27 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.19-openshift-4.13 AS rhel9-builder
 WORKDIR /go/src/github.com/openshift/sdn
 COPY . .
 RUN make build --warn-undefined-variables
-RUN CGO_ENABLED=0 GO_BUILD_FLAGS="-tags no_openssl" make build GO_BUILD_PACKAGES="github.com/openshift/sdn/cmd/openshift-sdn-cni" --warn-undefined-variables
+RUN CGO_ENABLED=1 make build GO_BUILD_PACKAGES="github.com/openshift/sdn/cmd/openshift-sdn-cni" --warn-undefined-variables
+
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13 AS rhel8-builder
+WORKDIR /go/src/github.com/openshift/sdn
+COPY . .
+RUN make build --warn-undefined-variables
+RUN CGO_ENABLED=1 make build GO_BUILD_PACKAGES="github.com/openshift/sdn/cmd/openshift-sdn-cni" --warn-undefined-variables
 
 FROM registry.ci.openshift.org/ocp/4.13:cli AS cli
 FROM registry.ci.openshift.org/ocp/4.13:base
 
-COPY --from=builder /go/src/github.com/openshift/sdn/openshift-sdn-node /usr/bin/
-COPY --from=builder /go/src/github.com/openshift/sdn/openshift-sdn-controller /usr/bin/
-COPY --from=builder /go/src/github.com/openshift/sdn/openshift-sdn-cni /opt/cni/bin/openshift-sdn
-COPY --from=builder /go/src/github.com/openshift/sdn/host-local /usr/bin/cni/osdn-host-local
+mkdir -p /opt/cni/bin/rhel9
+COPY --from=rhel9-builder /go/src/github.com/openshift/sdn/openshift-sdn-cni /opt/cni/bin/rhel9/openshift-sdn
+COPY --from=rhel9-builder /go/src/github.com/openshift/sdn/host-local /opt/cni/bin/rhel9/osdn-host-local
```

Comment:
@abhat turns out we don't need osdn-host-local to be built for RHEL9 since it's just run from inside the container.

But for RHEL8 we do want to keep it in /usr/bin/cni since that's where it gets execed from.